Cargo.toml

```diff
@@ -21,10 +21,10 @@ bevy_ui = ["bevy/bevy_ui", "bevy/bevy_text", "bevy/bevy_render"]
 
 [dependencies]
 interpolation = "0.2"
-bevy = { version = "0.7", default-features = false }
+bevy = { version = "0.8", default-features = false, features = ["bevy_asset"] }
```

Comment:
`bevy_asset` should have its own separate feature so users aren't forced to enable it. Probs not worth blocking the upgrade for this so created